### PR TITLE
Implement support for <img srcset="...">

### PR DESCRIPTION
**Pull request by lbguilherme**
It will collect dependencies from ``, just like already happens with ``.

### CODE REVIEW

**Comment by lbguilherme** [comment]
The test failure on CI seems unrelated to this PR

File: src/assets/HTMLAsset.js [hunk 1]
```diff
@@ -41,6 +41,28 @@ class HTMLAsset extends Asset {
...
+              }
+              pair[0] = assetPath;
+
+              newSources.push(pair.join(' '));
+            }
+
+            node.attrs[attr] = newSources.join(',');
+            this.isAstDirty = true;
+
+            continue;
```
Comment (devongovett) [inline src/assets/HTMLAsset.js]: Could we move this logic for parsing srcset to a separate function? Maybe we could have a map somewhere of custom handlers for certain attrs?

File: src/assets/HTMLAsset.js [hunk 2]
```diff
@@ -41,6 +41,28 @@ class HTMLAsset extends Asset {
...
+              const pair = source.trim().split(' ');
+              if (pair.length === 0) continue;
+
+              let assetPath = this.addURLDependency(
+                decodeURIComponent(pair[0])
+              );
+              if (!isURL(assetPath)) {
+                assetPath = urlJoin(this.options.publicURL, assetPath);
+              }
+              pair[0] = assetPath;
```
Comment (devongovett) [inline src/assets/HTMLAsset.js]: The logic of the above few lines could be shared with the copy below which was already there. Maybe make a function for it?

Comment (lbguilherme) [reply src/assets/HTMLAsset.js]: Done.

File: src/assets/HTMLAsset.js (hunk 1 above)
Comment (lbguilherme) [reply src/assets/HTMLAsset.js]: I moved this logic to its own function. About custom handlers, perhaps its better to wait until there is at least another one?

**Pull request merged**